index.html

```diff
@@ -141,31 +144,33 @@ <h2>About Myself</h2>
             </div>
```

Comment:
- [ ] `optional`
- [ ] Kindly the social icons as the figma template please the screenshots below
`figma template`
![Screenshot from 2022-07-07 10-17-14](https://user-images.githubusercontent.com/66230306/177875203-d8eb1929-a2df-4a10-8177-f6783e9170a5.png)

`your template`
![Screenshot from 2022-07-07 10-36-39](https://user-images.githubusercontent.com/66230306/177875509-25741258-be31-4050-9db2-52ab9f54ab2a.png)